server/webhook.go

```diff
@@ -150,8 +145,19 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusInternalServerError, err).SetInternal(err)
 		}
 
-		//nolint:misspell
-		// TODO(unknwon): Validate webhook signature, https://linear.app/bbteam/issue/BYT-935
+		body, err := io.ReadAll(c.Request().Body)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusBadRequest, "Failed to read webhook request").SetInternal(err)
+		}
```

Comment:
Since we can check the push event type through HTTP header, we don't need to eagerly read the request body.